new-examples/jdbc-atomic-batch-execute-operation/jdbc_atomic_batch_execute_operation.out

```diff
@@ -0,0 +1,18 @@
+# To run this sample, Create a ballerina project, copy the `jdbc_atomic_batch_execute_operation.bal`
+#file to the project home directory, add dependency jar details to `Ballerina.toml` and execute the `bal run` command below.
+bal run
```

Comment:
```suggestion
# Create a Ballerina project.
# Copy the example to the project and add relevant database driver jar details to the `Ballerina.toml` file.
# Execute the command below to build and run the project.
`bal run`
```

Reply:
@lahirusamith shall we follow this pattern in all BBEs

Reply:
@niveathika WDYT?

Reply:
added in https://github.com/ballerina-platform/ballerina-distribution/pull/1751/commits/8d5cd6b6cc1d1adae5eaf10e5ecf57a8bd4b4a7c